src/components/Dropdown/Dropdown.tsx

```diff
@@ -61,6 +66,7 @@ export class Dropdown extends React.Component<DropdownProps, State> {
     placement: 'bottom',
     mobileNativeSelect: false,
     upgrade: false,
+    optionsContainerId: 'dropdown-options-container',
```

Comment:
I'm not fully complete with the default value, but it has to be here to give the options dom id in order to put aria-activedescendant.
Do you think we should add something random here?
@jonathanadler

Reply:
yeah, we should add a random suffix
or add a static `_index` property that keeps count or something like this.
I think that, soon, when we publish `v2`, we can just require our consumers to use this `id` prop